src/debugger/dace_debugging_session.ts

```diff
@@ -38,18 +46,19 @@ export class DaceDebuggingSession extends LoggingDebugSession {
     protected launchRequest(
         response: DebugProtocol.LaunchResponse,
         args: DaceLaunchRequestArguments
-    ): Thenable<void> | void {
+    ): void {
         if (!this.folder) {
-            let msg = "Working folder not found, open a folder and try again";
-            return vscode.window.showErrorMessage(msg).then((_) => {
+            const msg = 'Working folder not found, open a folder and try again';
+            vscode.window.showErrorMessage(msg).then((_) => {
                 this.sendEvent(new TerminatedEvent());
                 return; // abort launch
             });
+            return;
         }
 
-        const buildType = !args.buildType ? "Debug" : args.buildType;
+        const buildType = args.buildType ?? 'Debug';
         const daceDev = args.daCeDev;
-        const portNum: string = String(PORT);
+        const portNum: string = String(DACE_DEBUG_PORT);
```

Comment:
does the debug functionality still work?

Reply:
No the debug functionality was removed from the UI around [1.5 years ago](https://github.com/spcl/dace-vscode/pull/260/files#diff-3bdabd7004490ac675177fb46f58afe062d97aa8d98b676cd9bdcdc51e267e45L145), but in working on this PR, one of the items that has made it onto the issues list for the upcoming weeks, is a PR to revive this functionality ;-)